### ISSUE TITLE
Code in NPM registry is not latest

### ISSUE THREAD

**Issue by seriflafont** [sha256/Pristine#50]
The js that is downloaded via npm install does not match the latest in the repo... can this please be fixed? The 'equals' and the 'pattern' both have issues in the version in the registry but they work great on the demos using the most current code. Our project relies on the registry as part of the build process for 3rd party code. Thanks in advance!

**Comment by t0byman** [comment]
Yes, please fix this. When I download your package via NPM the addMessages and setLocale functions aren't in there!

**Comment by zecompadre** [comment]
still missing latest version on NPM and jsdelivr

**Comment by kevinmu17** [comment]
Here's a "fix" ` "pristinejs": "git+https://github.com/sha256/Pristine.git"` Use this in your package.json and rerun `npm install`

@sha256 you left this repo for dead?

**Comment by manifestinteractive** [comment]
Agreed.  Looks like the author just forgot to bump their NPM package version in package.json, which is why we are getting the old code.  The README was correctly updated though, making it reference functions in NPM on the NPM website that don't actually exist.  Must use @kevinmu17 solution until resolved.
